.storybook/main.js

```diff
@@ -1,17 +1,123 @@
+                  cacheDirectory: `.cache/storybook`,
+                  presets: [
+                    [
+                      '@babel/preset-env',
+                      {
+                        shippedProposals: true,
+                        useBuiltIns: 'usage',
+                        corejs: 3,
+                      },
+                    ],
+                    '@babel/preset-typescript',
+                    IS_PRODUCTION && [
+                      'babel-preset-minify',
+                      { builtIns: false, mangle: false },
+                    ],
+                    '@babel/preset-react',
+                    '@babel/preset-flow',
+                  ].filter(Boolean),
+                  plugins: [
+                    '@babel/plugin-proposal-object-rest-spread',
+                    '@babel/plugin-proposal-class-properties',
+                    '@babel/plugin-syntax-dynamic-import',
+                    [
+                      'babel-plugin-emotion',
+                      { sourceMap: true, autoLabel: true },
+                    ],
+                    'babel-plugin-macros',
+                    'babel-plugin-add-react-displayname',
+                    [
+                      'babel-plugin-react-docgen',
+                      { DOC_GEN_COLLECTION_NAME: 'STORYBOOK_REACT_CLASSES' },
+                    ],
+                  ],
+                },
+              },
+            ],
+            exclude: [/node_modules/, /dist/],
+          },
+          {
+            test: /\.scss$/,
+            use: ['style-loader', 'css-loader', 'sass-loader'],
+            include: path.resolve(__dirname, '../src/main.scss'),
+          },
+          {
+            test: /\.svg$/,
+            loader: 'svg-sprite-loader',
+            include: path.join(SOURCE_DIR, 'images'),
+            options: {
+              symbolId: (filePath) => {
+                const pathParts = filePath.split(path.sep);
+                const symbol = path.basename(filePath, '.svg');
+
+                switch (pathParts[pathParts.length - 2]) {
+                  case 'math-symbols':
+                    return `sg-math-symbol-icon-${symbol}`;
+                  case 'icons':
+                    return `icon-${symbol}`;
+                  case 'subjects':
+                    return `icon-subject-${symbol}`;
+                  case 'subjects-mono':
+                    return `icon-subject-mono-${symbol}`;
+                  case 'mobile-icons':
+                    return `icon-mobile-${symbol}`;
+                  default:
+                    return symbol;
+                }
+              },
+            },
+          },
+          {
+            test: /\.(ico|jpg|jpeg|png|gif|eot|otf|webp|ttf|woff|woff2|cur|ani)(\?.*)?$/,
+            loader: 'file-loader',
+            query: {
+              name: 'static/media/[name].[hash:8].[ext]',
+            },
+          },
+        ],
+      },
+    };
+
+    _config.resolve.modules.push(
+      ...[
```

Comment:
Likely you don't need this spread here and you can just do:
```
_config.resolve.modules.push(
        SOURCE_COMPONENTS_DIR,
        SOURCE_DOCS_DIR,
        path.join(SOURCE_DIR, 'images'),
        'node_modules',
    );
```

Reply:
I think I cannot, push takes comma separatet args, if I push array it will be array in array
![image](https://user-images.githubusercontent.com/8572321/97012714-746c1100-1548-11eb-970f-9f2c426c5cc3.png)

Reply:
right